### PR TITLE
Ses Throttle

### PR DESCRIPTION
**Pull request by Wittiest**
This PR has been tested and does the following:
- Allows users to edit a config.rb file for email configuration and input files
- Automatically handles AWS SES rejection and resends emails
- Logs each batch of sent emails

The program is set to raise an error and to stop if we encounter an error besides throttling.

### CODE REVIEW

File: emailScript/src/send.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+require_relative '../utils/utils'
+
+$client = Aws::SES::Client.new(region: 'us-west-2')
+
+def generate_bulk_template
+  {
+    source: $config[:from],
+    template: $config[:template_name],
+    destinations: [],
+    default_template_data: {
```
Comment (nakamolto) [inline emailScript/src/send.rb:14]: I could see a situation where we have a first name or a last name in a csv but not both. I'm concerned we will end up with the following example:
First name: "Mike"
Last name: ""
Result: "Hello Mike supporter!"

What if we just made the first name "Valued supporter" and last name "" as default? I think that would cover 99% of edge cases.... can't think of too many times you'd only want to write someone's last name in an email.

Comment (nakamolto) [reply emailScript/src/send.rb:14]: @Wittiest

**Review by nakamolto — COMMENTED** [review]
good PR, see my one comment

**Comment by Wittiest** [comment]
This PR has been modified based on your advice @mikebarile

**Review by nucat176 — APPROVED** [review]

**Pull request merged**